packages/bulkapi/src/database/InitializedMapToArray.ts

```diff
@@ -1,22 +1,21 @@
 export class InitializedMapToArray<KEY, VALUE> {
     /**
-     * Use the stringified version of the KEY, so two objects with the same contents are identical
+     * Use the stringified version of the KEY, so two objects with the same contents are identical.
      */
     private map: Map<string, VALUE[]> = new Map<string, VALUE[]>()
-    
-    constructor() {
-    }
-    
+
+    constructor() {}
+
     get(key: KEY): VALUE[] {
         const keyString = JSON.stringify(key)
         let existingValue = this.map.get(keyString)
         if (existingValue === undefined) {
-           existingValue = []
-           this.map.set(keyString, existingValue) 
+            existingValue = []
```

Comment:
Isn't this a weird indentation (3 spaces)?

Reply:
Reformatted